npm/rules.bzl

```diff
@@ -83,39 +83,37 @@ assemble_npm = rule(
 )
 
 
-def _deploy_npm(ctx):
+def _deploy_npm_impl(ctx):
```

Comment:
Explanation:

1. Declare that we're creating a (shell) script to deploy an NPM package
2. Create the script from the template, `templates/deploy.py`, using variable substitution
3. When `bazel run` is invoked, run the script, passing the package to be deployed, the Java JAR containing the deployer code, and `npm`, and symlink `deploy_npm.tgz` to the package being deployed for convenience